plugins/wazuh-core/public/services/state/types.ts

```diff
@@ -0,0 +1,32 @@
+import { LifecycleService } from '../types';
+import { Subscription, BehaviourSubject } from 'rxjs';
+
+export interface StateContainer<T = any> {
```

Comment:
The state containers can be defined in different plugins, and due to the convention to expose services from a plugin through the plugin lifecycle instead of static imports, that approach could overcomplicate the definition of a new state.